src/app/src/main/kotlin/org/vaccineimpact/orderlyweb/app_start/routing/web/WebWorkflowRouteConfig.kt

```diff
@@ -9,9 +9,13 @@ object WebWorkflowRouteConfig : RouteConfig
     private val runReports = setOf("*/reports.run")
 
     override val endpoints: List<EndpointDefinition> = listOf(
-        WebEndpoint("/workflows", WorkflowRunController::class, "getWorkflowRunSummaries")
-            .json()
-            .secure(runReports)
-            .transform()
+            WebEndpoint("/workflows/:key", WorkflowRunController::class, "getRunWorkflowDetails")
```

Comment:
As previously commented, this should be renamed for consistency:
```suggestion
            WebEndpoint("/workflows/:key", WorkflowRunController::class, "getWorkflowRunDetails")
```